src/config/oracle-linux-addons.repo

```diff
@@ -0,0 +1,7 @@
+[oracle-linux-addons]
+name=Oracle Linux Addons
+baseurl=http://yum.oracle.com/repo/OracleLinux/OL8/addons/x86_64
+enabled=1
+gpgcheck=0
+repo_gpgcheck=0
```

Comment:
I'll gpgcheck=1 after I'll see this can be built by ci
repo_gpgcheck failed for me locally